docs/book/mlops-stacks/annotators/label-studio.md

```diff
@@ -0,0 +1,193 @@
+stacks will currently *not* work if you want to do add the annotation stack
+component as part of your stack.
+
+
+Feast integration currently supports your choice of offline data sources, and a
+[Redis](https://redis.com/) backend for your online feature serving. We
+encourage users to check out [Feast's documentation](https://docs.feast.dev/)
+and [guides](https://docs.feast.dev/how-to-guides/) on how to set up your
+offline and online data sources via the configuration `yaml` file.
+
+{% hint style="info" %} COMING SOON: The Label Studio Integration supports the
+use of annotations in an ML workflow, but we do not currently handle the
+universal conversion between data formats as part of the training workflow. Our
+initial use case was built to support image classification and object detection,
+but we will add helper steps and functions for other use cases in due course. We
+will update the docs when we enable this functionality. {% endhint %}
+
+## How to deploy it?
+
+The Label Studio Annotator flavor is provided by the Label Studio ZenML
+integration, you need to install it, to be able to register it as an Annotator
+and add it to your stack:
+
+```shell
+zenml integration install label_studio
+```
+
+Before registering a `label_studio` flavor stack component as part of your
+stack, you'll need to have registered a cloud artifact store and a secrets
+manager to handle authentication with Label Studio as well as any secrets
+required for the Artifact Store. (See the docs on how to register and [setup a
+cloud artifact store](../artifact-stores/artifact-stores.md) as well as [a
+secrets manager](../secrets-managers/secrets-managers.md).)
+
+Be sure to register an secret schema for whichever artifact store you choose,
+and then you should make sure to pass the name of that secret into the artifact
+store as the `--authentication_secret` as [described in this
+guide](../artifact-stores/amazon-s3.md#advanced-configuration), for example in
+the case of AWS.
+
+You will next need to obtain your Label Studio API key. This will give you
+access to the web annotation interface.
+
+```shell
+# choose a username and password for your label-studio account
+label-studio reset_password --username <username> --password <password>
+# start a temporary / one-off label-studio instance to get your API key
+label-studio start -p 8094
+```
+
+Then visit [http://localhost:8094/](http://localhost:8094/) to log in, and then
+visit [http://localhost:8094/user/account](http://localhost:8094/user/account)
+and get your Label Studio API key (from the upper right hand corner). You will
+need it for the next step. `Ctrl-c` out of the Label Studio server that is
+running on the terminal.
+
+At this point you should register the API key with your secrets manager under a
+custom secret name, making sure to replace the two parts in `<>` with whatever
+you choose:
+
+```shell
+zenml secret register <label_studio_secret_name> --api_key="<your_label_studio_api_key>"
```

Comment:
I personally put the placeholders in capital letter to make them stick out even more, not sure if we have any convention on that though

Reply:
Done.